addons/web/static/src/views/fields/statusbar/statusbar_field.xml

```diff
@@ -2,6 +2,32 @@
 <templates xml:space="preserve">
 
     <t t-name="web.StatusBarField" owl="1">
+        <t t-call-assets="web.assets_common"/>
+        <t t-if="this.props.showTimeOption &amp;&amp; this.state.showTimeState">
+            <button
+                type="button"
+                class="btn btn-secondary"
+                t-on-click="changeTimeState"
+                data-hotkey="t"
+                data-tooltip="Hide time spent"
+                aria-label="Hide time spent"
+            >
+                <i class="fa fa-calendar-times-o fa-fw"/>
+            </button>
+        </t>
+        <t t-if="this.props.showTimeOption &amp;&amp; !this.state.showTimeState">
+            <div class="time-container-hover">
+                <button
+                    type="button"
+                    class="btn btn-secondary time-btn-hover"
+                    t-on-click="changeTimeState"
+                    data-hotkey="t"
+                    data-tooltip="Show time spent"
+                    aria-label="Show time spent">
+                    <i class="fa fa-calendar fa-fw"/>
+                </button>
+            </div>
+        </t>
```

Comment:
You can put the `t-if` on the button and the div respectively. Since you don't have multiple components in there.

Not sure why one has a div and not the other too. Maybe it can just be 1 button with template values?
``<t t-set="showTime" t-value="your_condition">
 <button t-att-class="{'time-btn-hover': showTime}"
``
And such